### ISSUE TITLE
ApiTestCase does not work on Windows

### ISSUE THREAD

**Issue by cevou** [kleijnweb/swagger-bundle#82]
There is a hard coded path in the `ApiTestCase` class:

```php
file_put_contents(
    '/tmp/swagger.json',
    json_encode(self::$document->getDefinition(), JSON_PRETTY_PRINT)
);
```

The path to `/tmp/swagger.json` cannot be resolved on Windows.

**Comment by kleijnweb** [comment]
Obviously that statement should not be, totally obsolete. It was already removed in master.  I removed the statement in the 3.x branch as well. I will release a new patch version later today, in the mean time you can either remove the statement from your vendor directory or `composer require kleijnweb/swagger-bundle=dev-3.x`.

Having said that, please note that SwaggerBundle tested nor guaranteed to run properly on Windows, something which is not likely to change. I'll gladly fix minor issues reported though :)

**Comment by kleijnweb** [comment]
This has been included in [v3.1.3](https://github.com/kleijnweb/swagger-bundle/releases/tag/v3.1.3) which is now latest stable.

**Comment by cevou** [comment]
Thanks for fixing that. I already removed the code from the vendor directory and it worked after that.
Other than that everything works fine on Windows. I only use it on one of my PCs for development, so while it is not a problem to run tests in Travis, I didn't like that I was not able to execute them during development.
Great that it works now!
